readme.txt

```diff
@@ -1,9 +1,9 @@
 === GoDaddy Email Marketing ===
-Contributors: godaddy, madmimi, humanmade
+Contributors: godaddy, madmimi, humanmade, lukecarbis, marcin.lawrowski, xwp
 Tags: forms, newsletter, opt-in, email marketing, signup, subscribe
 Requires at least: 3.5
-Tested up to: 4.3
-Stable tag: 1.0.3
+Tested up to: 4.4.2
```

Comment:
@marcin-lawrowski let's go ahead and bump this to 4.5 since the builds are passing on trunk.